packages/svelte/src/internal/client/render.js

```diff
@@ -143,6 +160,9 @@ export function hydrate(component, options) {
 		}, false);
 	} catch (error) {
 		if (error === HYDRATION_ERROR) {
+			// TODO it's possible for event listeners to have been added and
+			// not removed, e.g. with `<svelte:window>` or `<svelte:document>`
```

Comment:
a drive-by observation you didn't want to address as part of this PR because unrelated?

Reply:
yeah — just opened https://github.com/sveltejs/svelte/issues/12360